### DISCUSSION TITLE
Google oidc offline_access not working?

### DISCUSSION THREAD

**Discussion by nxy7** [ory/kratos#2472] [Q&A] [answered]
Hello, I'm trying to get refresh token from google, but it seems like "offline_access" doesn't work. In docs 'configuration' page offline access is set up with 'scopes', but that doesn't seem to work.
<img width="320" alt="image" src="https://user-images.githubusercontent.com/32909532/168490125-429c95e6-d66d-4fad-839c-716287461854.png">
According to google docs offline access is provided by specyfing access_type=offline in request and it's not bundled with scopes.
Is there something im missing?

**Comment by vinckr** [comment, marked as answer]
Hello @nxy7 
apologies for the late answer!
Were you able to figure it out? Is offline_access actually part of your scopes or do you just specify access_type=offline in the request?
